open_city_profile/tests/conftest.py

```diff
@@ -107,8 +107,7 @@ def mock_responses():
 
 
 def unix_timestamp_now():
-    epoch = datetime(1970, 1, 1, tzinfo=timezone.utc)
-    return int((datetime.now(tz=timezone.utc) - epoch).total_seconds())
+    return int(datetime.now(tz=timezone.utc).timestamp())
```

Comment:
Much better than calculating this by ourselves.

Wouldn't this be much clearer if this was just
```python
@pytest.fixture
def unix_timestamp_now():
    return int(datetime.now(tz=timezone.utc).timestamp())
```

Would be more in line with the other fixtures too. As usually the function is the same name as the fixture.

If there is a compelling reason why the timestamp generation should be separate, it could be better to rename the functions. Something like

```python
def get_unix_timestamp_now():
    return int(datetime.now(tz=timezone.utc).timestamp())

@pytest.fixture
def unix_timestamp_now():
    return get_unix_timestamp_now()
```

Reply:
The unix timestamp provider is also used as a function (in places where fixtures can't be used) so there's a need for both the function and the fixture. Changed as you proposed in 96359d9. I agree it's cleaner this way.